node/node.go

```diff
@@ -101,9 +102,17 @@ func (n *Node) Start(ctx context.Context) error {
 
 	// TODO(@Wondertan): Print useful information about the node:
 	//  * API address
-	//  * Pubkey/PeerID
 	//  * Host listening address
 	log.Infof("started %s Node", n.Type)
+
+	addrs, err := peer.AddrInfoToP2pAddrs(host.InfoFromHost(n.Host))
+	if err != nil {
+		log.Errorf("Error while retreiving multiaddress information: %s", err)
+		return err
+	}
+	for _, addr := range addrs {
```

Comment:
Pls, don't print duplicates, so you just need to print the second half `addrs[len(addrs)/2:]`